style.css

```diff
@@ -0,0 +1,90 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,400;0,700;1,400;1,600&display=swap");
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+html,
+body {
+  background-color: #fff;
+  color: #000;
+  font-family: "Poppins", sans-serif;
+  width: 400px;
+  margin: auto;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+  cursor: pointer;
+}
+
+ul {
+  list-style: none;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  color: #6070ff;
+  margin: 10px;
+}
+
+.logo {
+  font-size: 18px;
+  font-weight: 700;
+  line-height: 20px;
+  letter-spacing: 0;
+  text-align: center;
+  margin-top: 10px;
+  margin-left: 12px;
+}
+
+.main-section {
+  background-image: url(./assest/img/header-shapes\ mobile.jpg);
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
```

Comment:
- [ ] [OPTIONAL] Kindly align your `background-image` to display as seen in your chosen Figma design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/205439707-1f51610e-ffeb-4af9-91e0-35a8da854939.png)  |  ![image](https://user-images.githubusercontent.com/74814780/205439699-5b8d11ab-b69d-4b3e-a8e3-98ea4c8ce019.png)